### PR TITLE
SWATCH-124: Deploy swatch-subscriptions-sync via its own clowdapp

### PR DESCRIPTION
**Pull request by lindseyburnett**
https://issues.redhat.com/browse/SWATCH-124

```bash
export NAMESPACE=$(bonfire namespace reserve -d 2h)
```

```bash
bonfire deploy \
  -n $NAMESPACE \
  -i quay.io/cloudservices/rhsm-subscriptions=1d2ddc8 \
  -i quay.io/cloudservices/swatch-system-conduit=1d2ddc8 \
  -i quay.io/cloudservices/swatch-producer-aws=1d2ddc8 \
  --optional-deps-method=none \
  --no-remove-resources=rhsm \
  --no-remove-resources=swatch-subscription-sync \
  --no-remove-resources=swatch-metrics \
  --no-remove-resources=swatch-producer-red-hat-marketplace \
 rhsm
```

If you're having resource issues, try removing some of the less-than-helpful ClowdApps that deploy whether you want them to or not.  I usually blow up `xjoin-search`, `ingress`, and `puptoo`.

```bash
oc project $NAMESPACE
oc get clowdapp
```
```bash
oc delete clowdapp xjoin-search
oc delete clowdapp puptoo
oc delete clowdapp ingress
```

### CODE REVIEW

File: deploy/rhsm-clowdapp.yaml [hunk 1]
```diff
@@ -1828,310 +1825,7 @@ objects:
             - name: logs
               emptyDir:
 
-      - name: subscription-sync
```
Comment (kahowell) [inline deploy/rhsm-clowdapp.yaml:1831]: Should also remove `capacity-ingress`

**Review by kahowell — CHANGES_REQUESTED** [review]

**Review by mstead — APPROVED** [review]
Approve pending pr_checks passing.

**Pull request merged**